lib/positioning/healer.rb

```diff
@@ -8,7 +8,7 @@ def initialize(model, column, order)
 
     def heal
       if positioning_columns.present?
-        @model.select(*positioning_columns).distinct.each do |scope_record|
+        @model.unscope(:order).select(*positioning_columns).distinct.each do |scope_record|
```

Comment:
Actually, we should also change `select` to `reselect` so that it blasts away any cases where people have a default scope involving `select`.